packages/node/src/models/state-channel.ts

```diff
@@ -207,22 +197,18 @@ export class StateChannel {
     return this.getSigningKeysFor(this.monotonicNumInstalledApps);
   }
 
-  public getETHFreeBalance(): AppInstance {
-    if (!this.freeBalanceAppIndexes.has(0)) {
-      throw Error(ERRORS.FREE_BALANCE_MISSING);
-    }
-
-    const idx = this.freeBalanceAppIndexes.get(0);
-
-    if (idx === undefined || !this.appInstances.has(idx)) {
-      throw Error(ERRORS.FREE_BALANCE_IDX_CORRUPT(idx!));
+  public getFreeBalanceAppInstance(): AppInstance {
```

Comment:
Make this a getter so other internal retrievals would throw this error (instead of doing `!`) if no app instance is set for the free balance.